boomerang.js

```diff
@@ -1534,7 +1539,13 @@ BOOMR_check_doc_domain();
 					}
 					xhr.setRequestHeader (impl.beacon_auth_key, impl.beacon_auth_token );
 				}
-				xhr.send(paramsJoined);
+				// If the user has opted to post as JSON, then create JSON from the parameters
+				//     otherwise send it as parameters separated by & 
+				if(impl.beacon_post_format !== ""  && impl.beacon_post_format === "JSON") {
```

Comment:
you will also need to check that the user agent has `JSON` available, and if you're going to do JSON, then there's a whole separate path to take.  We intentionally removed this support after running into many problems.  I'd recommend undoing this change and going with what I approved yesterday.